src/utils/coingecko.ts

```diff
@@ -30,10 +30,10 @@ const NATIVE_TOKEN_IDS: Partial<Record<Chain, string>> = {
   Worldchain: 'ethereum',
   Seievm: 'sei',
   Mezo: 'wrapped-bitcoin',
+  HyperEVM: 'hyperliquid',
   Plume: 'plume',
   Ink: 'ethereum',
-  HyperEVM: 'hyperliquid',
-  HyperCore: 'usd-coin',
+  CreditCoin: 'creditcoin',
```

Comment:
I think it is `creditcoin-2`

Reply:
Let's use https://www.coingecko.com/en/coins/wrapped-ctc @DBosley 

After discussion with their team, they'll OTC us any necessary CTC at this price.